.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Nginx image creator
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+
+  create-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    steps:
+    - name: CHECKOUT
+      uses: actions/checkout@v3
+
+    - name: INSTALL Node
+      working-directory: ./node
+      run: npm install
+
+    - name: RUN SuperTest API Testing
+      working-directory: ./node
+      run: npm test
+
+    - name: BUILD Angular
+      working-directory: ./angular
+      run: |
+        npm link @angular/cli
+        ng build
+
+    - name: LOGIN to the Container registry
+      uses: docker/login-action@v3.3.0
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Let's use Github Actions automatic variables?
https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication
![image](https://github.com/user-attachments/assets/03746fff-1ba7-4b7d-9bfa-1a9df02c224d)